### ISSUE TITLE
Model loading is disabled in --multi-user mode l

### ISSUE THREAD

**Issue by ppsk1ng** [oobabooga/text-generation-webui#4356]
### Describe the bug

![image](https://github.com/oobabooga/text-generation-webui/assets/48208176/5acc92ad-eed3-4291-9002-bf06e3d41b50)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Simpy run the code with --multi-user option.

### Logs

```shell
(base) [liaowang@vam1-rcc-model-training-gpu1] /data0/liaowang/Downloads/text-generation-webui-main$ python server.py --listen --verbose --multi-user --model-menu
2023-10-22 12:48:06 WARNING:
You are potentially exposing the web UI to the entire internet without any access password.
You can create one with the "--gradio-auth" flag like this:

--gradio-auth username:password

Make sure to replace username:password with your own.
2023-10-22 12:48:06 WARNING:
The multi-user mode is highly experimental and should not be shared publicly.
2023-10-22 12:48:08 INFO:Loading settings from settings.yaml...
The following models are available:

1. Llama-2-7b-chat-hf
2. None
3. ShiningValiant-1-2-AWQ

Which one do you want to load? 1-3

3
```

### System Info

```shell
Linux
```

**Comment by drc-pnnl** [comment]
This is still a problem. If you start with --multi-user you can't load a model.  In the "model" tab you are unable to click in any of the fields to either load a model or other settings.

**Comment by Trustmaster554** [comment]
I have the same issue.

**Comment by NathanJThorpe** [comment]
Same problem here.

**Comment by kyleboddy** [comment]
Same problem. @ppsk1ng 

These areas where I drew on the screen are simply X'd out and disabled.

![image](https://github.com/oobabooga/text-generation-webui/assets/746351/7dbe17be-b5f8-4944-a3ac-b3f991502f80)

Console:

```
kyle@aeternum:~/text-generation-webui$ ./start_linux.sh --listen --multi-user
19:54:41-270592 INFO     Starting Text generation web UI
19:54:41-273410 WARNING
                         You are potentially exposing the web UI to the entire internet without any
                         access password.
                         You can create one with the "--gradio-auth" flag like this:

                         --gradio-auth username:password

                         Make sure to replace username:password with your own.
19:54:41-274677 WARNING
                         The multi-user mode is highly experimental and should not be shared publicly.
19:54:41-275814 INFO     Loading the extension "gallery"
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
```

**Comment by emitrokhin** [comment]
Same Problem when I run the code with --multi-user option.

**Comment by bangkokpadang** [comment]
I just wanted to add that at least for now, right clicking each of the buttons/elements, selecting, "inspect element" and then deleting the css class of "disabled" enables the fields and buttons. The only caveat is that you need to manually type in the exact name of the model file (or folder for GPTQ/EXL2/other model types). It's probably just a bug in the stylesheet somewhere. I just tested this dirty "fix" and its working again with 2 simultaneous users from 2 separate devices/networks.

**Comment by kdubey22** [comment]
Same problem @ppsk1ng 
@kyleboddy

**Comment by kyleboddy** [comment]
> I just wanted to add that at least for now, right clicking each of the buttons/elements, selecting, "inspect element" and then deleting the css class of "disabled" reenables the fields and buttons. The only caveat is that you need to manually type in the exact name of the model file (or folder for GPTQ/EXL2/other model types). It's probably just a bug in the stylesheet somewhere. I just tested this dirty "fix" and its working again with 2 simultaneous users from 2 separate devices/networks. Certainly not a solution, but might get the job done in a pinch.

Interesting, I'll give it a shot. I'm surprised to hear it's a simple CSS issue.

**Comment by BeHappy2021** [comment]
Same problem and removing "disabled" in the css works for me. I am wondering if this is intentional though so that the model can't be switched by any of the users.

**Comment by psliva** [comment]
@BeHappy2021 yes, my assumption when I ran into this was that you should provide the model load definition (and config parameters if applicable) in the CMD_FLAGS.txt file as well, e.x.

> --model miqu-1-120b-gptq
> --gpu-split 18,18,18,15

**Comment by G4Zz0L1** [comment]
Still relevant to date, any news on this?

**Comment by Nurgl** [comment]
This option is also interesting. is it possible to make multiple modes. Chatbot mode (chat only) and fully multi-user mode with the ability to upload multiple models. The first mode is easy to implement, you need to make it possible to hide the menu bar in css by a parameter in the config and make a queue counter, as I understand the problem is that the launchers cannot work asynchronously. The problem is that people have different expectations for different tasks from the multiplayer mode.
